content/swiftui/concepts/tables/tables.md

```diff
@@ -0,0 +1,118 @@
+---
+Title: 'Tables' 
+Description: 'A container that displays data in rows organized into one or more columns and optionally allows the selection of one or more items.'
+Subjects: 
+  - 'iOS'
+  - 'Mobile Development'
+Tags:
+  - 'iOS'
+  - 'SwiftUI'
+  - 'Tables'
+CatalogContent:
+  - 'learn-ios-course'
+  - 'paths/ios-developer'
+---
+
+
+**Table** in SwiftUI is a structure that allows us to create and display collections of data in columns and rows. Many datasets have a tabular structure and can be efficiently presented in a table format. If the number of rows exceeds the available space, `Table` will enable vertical scrolling automatically. Additionally, on macOS, horizontal scrolling is available when the number of columns exceeds the view's width. 
+
+## Creating a Table 
+
+`Table` is a great option for displaying a collection of data in a compact form. In the example below, in order to create a simple three-column table from an array of Singer your data model has to conform to `Identifiable`.
+
+### Example
```

Comment:
```suggestion
## Example

`Table` is an excellent choice for presenting data in a concise, organized manner. In the example below, to create a straightforward three-column table from an array of `Singer`, the data model needs to conform to `Identifiable`.

```
Let's remove the above heading and add it below the example heading only. Also refrain from using pronouns.